homeassistant/components/flux_led/light.py

```diff
@@ -139,41 +147,66 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+async def async_setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the platform and manage importing from YAML."""
+
+    if config.get("automatic_add", False):
+        await hass.config_entries.flow.async_init(
+            DOMAIN, context={"source": SOURCE_IMPORT}, data={CONF_TYPE: "auto"}
+        )
+
+    else:
+        for import_host, import_item in config.items():
```

Comment:
We want to iteration the `CONF_DEVICES` config item not the whole config.

Reply:
Is that:

`for import_host, import_item in config[CONF_DEVICES].items():`?

Reply:
Yes. :+1:

Reply:
OK - addressed.